### PR TITLE
Term entry c sharp compare

### LINKED ISSUE #1833: [Term Entry] C# Strings `.Compare()`
### Reason/inspiration (optional)

We would like a new term entry for [C#](https://www.codecademy.com/resources/docs/c-sharp) about the `String.Compare()` method.

The entry should go in a new file in docs/content/c-sharp/concepts/strings/terms/compare/compare.md

The entry should

- Have a syntax section illustrating the method with a ` ```pseudo ` code block.
- An example showing the method in use with a ` ```cs ` code block.
- A runnable codebyte example in a ` ```codebyte/csharp ` block.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by cslylla**
### Description

- Added new term for C# String methods: .Compare()
- Closes #1833 

### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/c-sharp/concepts/strings/terms/compare/compare.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+In the first case, `pizza` comes before `Pizza` in alphabetical order, as it is lowercase and the method is case-sensitive. In the second case, `Pizza` comes before `waffle` in alphabetical order. This example results in the following output:
+
+```shell
+1
+-1
+```
+
+## Codebyte Example
+
+In the following runnable example, the `.Compare()` method determines the alphabetical order of the given `string1` and `string2`, while the letter casing is ignored. Finally, the `WriteLine()` method prints the returned integer to the console.
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/strings/terms/compare/compare.md:59]:
```suggestion
In the following runnable example, the `.Compare()` method determines the alphabetical order of `string1` and `string2`, while the letter casing is ignored. Finally, the `WriteLine()` method prints the returned integer to the console.
```

File: content/c-sharp/concepts/strings/terms/compare/compare.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+```
+
+The `.Compare()` method is called on the `String` class. It takes two `string` type arguments, `string1` and `string2`, to compare them alphabetically.
+
+> **Note:** `ignoreCase` is an optional `boolean` type argument. By default, `false` or not specified, the `.Compare()` method is case-sensitive. Passing `true` as the third argument makes the`.Compare()` method non-case-sensitive.
+
+The `.Compare()` method returns an `integer` that represents the relative order of `string1` and `string2` in the alphabet:
+
+- `0` if their position is the same (`string1` and `string2` are identical)
+- a negative `integer` if `string1` comes before `string2`
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/strings/terms/compare/compare.md:30]:
```suggestion
- `-1` if `string1` comes before `string2`
```

File: content/c-sharp/concepts/strings/terms/compare/compare.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+- a positive `integer` if `string2` comes before `string1`
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/strings/terms/compare/compare.md:31]:
```suggestion
- `1` if `string2` comes before `string1`
```

File: content/c-sharp/concepts/strings/terms/compare/compare.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+## Syntax
+
+```pseudo
+String.Compare(string1, string2, `ignoreCase`);
+```
+
+The `.Compare()` method is called on the `String` class. It takes two `string` type arguments, `string1` and `string2`, to compare them alphabetically.
+
+> **Note:** `ignoreCase` is an optional `boolean` type argument. By default, `false` or not specified, the `.Compare()` method is case-sensitive. Passing `true` as the third argument makes the`.Compare()` method non-case-sensitive.
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/strings/terms/compare/compare.md:25]:
```suggestion
> **Note:** `ignoreCase` is an optional `boolean` type argument. By default the `.Compare()` method is case-sensitive. Passing `true` as the third argument makes the `.Compare()` method case insensitive.
```

File: content/c-sharp/concepts/strings/terms/compare/compare.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+The **`.Compare()`** method is a string method that compares two specified [`strings`](https://www.codecademy.com/resources/docs/c-sharp/strings) in alphabetical order and returns an integer that represents their relative order.
+
+## Syntax
+
+```pseudo
+String.Compare(string1, string2, `ignoreCase`);
+```
+
+The `.Compare()` method is called on the `String` class. It takes two `string` type arguments, `string1` and `string2`, to compare them alphabetically.
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/strings/terms/compare/compare.md:23]:
```suggestion
The `.Compare()` method is called on the `String` class. It takes two `string` type arguments, `string1` and `string2`,  and compares them alphabetically.
```

File: content/c-sharp/concepts/strings/terms/compare/compare.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
+---
+
+The **`.Compare()`** method is a string method that compares two specified [`strings`](https://www.codecademy.com/resources/docs/c-sharp/strings) in alphabetical order and returns an integer that represents their relative order.
+
+## Syntax
+
+```pseudo
+String.Compare(string1, string2, `ignoreCase`);
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/strings/terms/compare/compare.md:20]:
```suggestion
String.Compare(string1, string2, ignoreCase);
```

File: content/c-sharp/concepts/strings/terms/compare/compare.md [hunk 7]
```diff
@@ -0,0 +1,71 @@
+---
+Title: '.Compare()'
+Description: 'Compares two specified strings in alphabetical order and returns an integer that represents their relative order.'
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/strings/terms/compare/compare.md:3]:
```suggestion
Description: 'Compares the alphabetical order of two strings and returns an integer that represents their relative ranking.'
```

File: content/c-sharp/concepts/strings/terms/compare/compare.md [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+  - 'Code Foundations'
+Tags:
+  - 'Strings'
+  - 'Methods'
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
+---
+
+The **`.Compare()`** method is a string method that compares two specified [`strings`](https://www.codecademy.com/resources/docs/c-sharp/strings) in alphabetical order and returns an integer that represents their relative order.
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/strings/terms/compare/compare.md:15]:
```suggestion
The **`.Compare()`** method is a string method that compares the alphabetical order of two specified [`strings`](https://www.codecademy.com/resources/docs/c-sharp/strings) and returns an integer that represents their relative ranking.
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@cslylla I've completed my initial review. Please update the entry and tag me when you've finished making changes. Thanks

**Comment by cslylla** [comment]
@caupolicandiaz I accepted the changes.

**Review by caupolicandiaz — APPROVED** [review]
@cslylla  All set!

**Review by yangc95 — APPROVED** [review]

File: content/c-sharp/concepts/strings/terms/compare/compare.md [hunk 9]
```diff
@@ -0,0 +1,70 @@
...
+The **`.Compare()`** method is a string method that compares the alphabetical order of two specified [`strings`](https://www.codecademy.com/resources/docs/c-sharp/strings) and returns an integer that represents their relative ranking.
+
+## Syntax
+
+```pseudo
+String.Compare(string1, string2, ignoreCase);
+```
+
+- The `.Compare()` method is called on the `String` class. It takes two `string` type arguments, `string1` and `string2`,  and compares them alphabetically.
+- `ignoreCase` is an optional `boolean` type argument. By default the `.Compare()` method is case-sensitive. Passing `true` as the third argument makes the `.Compare()` method case insensitive.
```
Comment (yangc95) [inline content/c-sharp/concepts/strings/terms/compare/compare.md:24]:
```suggestion
- `ignoreCase` is an optional `boolean` type argument. By default the `.Compare()` method is case-sensitive. Passing `true` as the third argument makes the `.Compare()` method case-insensitive.
```

**Comment by yangc95** [comment]
🙌 Always a pleasure reviewing your contributions ☺️  @cslylla live link: https://www.codecademy.com/resources/docs/c-sharp/strings/compare

cc: @caupolicandiaz

**Pull request merged**